common/helpers/error_handlers.py

```diff
@@ -0,0 +1,31 @@
+import sys
+from django.shortcuts import redirect
+from common.helpers.constants import FrontEndSection
+from common.helpers.dictionaries import merge_dicts
+from common.helpers.front_end import section_url
+
+
+class ReportableError(Exception):
+    """Exception raised that needs to be logged and sent to a front end error page
+
+    Attributes:
+        message -- explanation of the error to be reported in the logs
+        front_end_args -- arguments to be surfaced on the front end error page
+    """
+
+    def __init__(self, message, front_end_args):
+        self.message = message
+        self.front_end_args = front_end_args or {}
+
+
+def handle500(request):
```

Comment:
Just one question- Till now there was no explicit error handler page as such in Front-End. With this change, any 500 error is redirected to this new Error page. Are we fine with that?

Reply:
Yes, as even a bare-bones error page is better than the blank error 500 page people were getting up until now.